### DISCUSSION TITLE
Lagrangian Particle Properties in FDS

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13392] [Q&A]
Hello, I would greatly appreciate your response. In my simulation of fire spread on a pine needle bed with a velocity inlet, I need to define the particle properties in the PART_ID by specifying the number of particles per cell. To do this, I calculated the volume of the bed as 𝑉=0.09×0.09×0.005 m^3 and used the bulk density formula 𝜌=𝑚/𝑉= 4g/4.05×10^5, which gave me approximately 1.782 particles per cell. I rounded this to 2 particles per cell and used my calculated bulk density for the mass per volume. My question is whether it is necessary to define both the number of particles per cell and the mass per volume, or if this is redundant in FDS. Can FDS calculate one of these parameters if only the other is provided in the input file?

&SURF ID                                   = 'Pine Needles'
      MATL_ID                               = 'GENERIC VEGETATION'
      MOISTURE_FRACTION         = 0.4117647059
      SURFACE_VOLUME_RATIO   = 8366.
      LENGTH                                = 0.07
      GEOMETRY                           = 'CYLINDRICAL' /

&PART ID                                    = 'Needles' 
      DRAG_COEFFICIENT             = 2.2
      COLOR                                 = 'GREEN'
      SURF_ID                               = 'Pine Needles' 
      QUANTITIES                        = 'PARTICLE TEMPERATURE','PARTICLE MASS'
      STATIC                                 =  .TRUE./

&INIT PART_ID                           = 'Needles'
      XB                                        = 0.255,0.345,0.085,0.175,0,0.005
      N_PARTICLES_PER_CELL       = 2
      CELL_CENTERED                  = .TRUE.
      MASS_PER_VOLUME           = 98.7654321
      DRY                                      = T /

**Comment by drjfloyd** [comment]
Particles in FDS do not necessarily represent just a single particle. Consider a residential type sprinkler flowing ~ 1 kg/s with 1 mm droplets.  This is ~2 million droplets per second. Simulating each droplet would take an enormous amount of computational time and memory.  Instead, we track a much smaller number of droplets and weight each droplet to preserve mass.  Rather than 2 million droplets per second you might track 2000 droplets per second and weight each drop with a factor of 1000.  The same applies for particles representing vegetation. The size of the particle on SURF gives the physical size of the particle but MPUV combined with N_PARTICLES_PER_CELL tells FDS how much to weight each particle. In your case you could just have N_PARTICLES_PER_CELL=1, and FDS would weigh each particle (in this case 1.782) to get the MPUV.

**Reply by SaharRasoulpour** [reply]
Thank you for your response, so it means that I don't necessarily need to specify both parameters. If I specify one (MASS_PER_VOLUME), FDS can internally calculate the necessary weight based on the number of particles defined per cell?

**Reply by drjfloyd** [reply]
It depends.  If you are modeling every particle and that gives you the exact mass, then you don't need to give information for weighting, each particle just gets a weight of 1.  If you are modeling some number of particles other than the actual number needed to achieve your mass, then you need to give both the number of particles and some input related to the total mass.  In your case the information you gave is that 2 particles is too much mass and 1 particle is too little mass. I would suggest 1 particle with the mass per volume so each particle would be weighted to give the correct mass.
